MinecraftClient/Settings.cs

```diff
@@ -204,7 +204,7 @@ private enum ParseMode { Default, Main, AppVars, Proxy, MCSettings, AntiAFK, Han
         /// <param name="settingsfile">File to load</param>
         public static void LoadSettings(string settingsfile)
         {
-            ConsoleIO.WriteLogLine(String.Format("[Settings] Loading Settings from {0}", System.IO.Path.GetFullPath(settingsfile)));
+            ConsoleIO.WriteLogLine(Translations.Get("mcc.setting", System.IO.Path.GetFullPath(settingsfile)));
```

Comment:
`[Settings]` is the class name actually so it should be kept as `[ClassName]` independently of the translation.